src/templateHandler.ts

```diff
@@ -100,6 +120,16 @@ export class TemplateHandler {
     // private methods
     //
 
+    private async callExtensions<T extends Binary>(extensions: TemplateExtension[], scopeData: ScopeData, context: TemplateContext): Promise<void> {
+        for (const extension of extensions) {
+            await extension.execute({
```

Comment:
First parameter (extension utilities) should no longer be part of `extension.execute`.